assets/app/views/_template.html

```diff
@@ -0,0 +1,7 @@
+<li class="template">
+  <img ng-src="{{template.icon}}">
```

Comment:
might handle it as a filter,  so it would be {{template | icon}}  and the icon filter can handle various different resource types and check if annotations.icon is defined, or return the default for that resource type if its not